app/api/routers/diagnoses.py

```diff
@@ -3,7 +3,7 @@
 from ..models import DataElementURI, VocabLabelsResponse
 from . import route_factory
 
-EXTERNAL_VOCAB = "snomed"
+EXTERNAL_VOCAB = "snomed_disorder"
```

Comment:
why is the vocab file already renamed in the main branch? Is this from a daisy chained PR?

Reply:
I believe it was called `snomedct_disorder` from the start.